ui/src/containers/SampleGridTableContainer.jsx

```diff
@@ -479,7 +482,7 @@ export default function SampleGridTableContainer(props) {
     e.stopPropagation();
   }
 
-  function isSingleCell() {
+  function getIsSingleCell() {
```

Comment:
No need to keep this function at all, just inline the logic above when initialising the `isSingleCell` variable.